### PR TITLE
Done with admin login

### PR DESCRIPTION
**Pull request by mounikonduru**
Did admin-login

### CODE REVIEW

File: admin/auth.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from flask import flash
+from flask_login import login_user
+
+def can_user_login(password, user, role):
```
Comment (chethana613) [inline admin/auth.py]: I don't see the changes of main file from where this can_user_login method being called. Did you miss some file commits to this PR?

**Review by chethana613 — COMMENTED** [review]
I don't see the changes of main file from where this can_user_login method being called in the PR. Did you miss some file commits to this PR?

File: admin/auth.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from flask import flash
+from flask_login import login_user
+
+def can_admin_login(password, user, role):
+    if user.check_password(password):
```
Comment (ramyagovind13) [inline admin/auth.py]: Please let me know what is user.check_password do?

Comment (ramyagovind13) [inline admin/auth.py]: Looks like something going wrong in the line:5, can you check it?

File: admin/inventory.py [hunk 3]
```diff
@@ -4,7 +4,6 @@
 
 def add(name, description, quantity, category, weight, expiry_date):
     try:
-        # Creating an instance of the Inventory class
```
Comment (ramyagovind13) [inline admin/inventory.py:7]: Can you add the commented line again? Let me know the reason why you have removed the commented line

File: admin/inventory.py [hunk 4]
```diff
@@ -13,7 +12,6 @@ def add(name, description, quantity, category, weight, expiry_date):
         weight=weight,
         expiry_date=expiry_date
         )               
-        # Adding the instance to the database session and commiting the changes
```
Comment (ramyagovind13) [inline admin/inventory.py:16]: same applies here too

File: website/__init__.py [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
-from flask_login import LoginManager #flask_login creates connection between the cookie and actual data in the DB
+from flask_login import LoginManager 
 
 db = SQLAlchemy()
 
 def create_app():
     app = Flask(__name__)
     app.config['SECRET_KEY'] = 'secretkey'
-    app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:abc@localhost:5432/shop_easy'
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:root@localhost:5432/shop_easy'
```
Comment (ramyagovind13) [inline website/__init__.py]: You should commit your password, change to postgres:password

File: admin/auth.py (hunk 2 above)
Comment (mounikonduru) [reply admin/auth.py]: check_password(password) is a method that is expected to check whether the provided password matches the stored password for the user.

Comment (mounikonduru) [reply admin/auth.py]: now i changed it to another one

File: website/__init__.py (hunk 5 above)
Comment (ramyagovind13) [reply website/__init__.py]: postgresql://postgres:password@localhost:5432/shop_easy

change to this

**Comment by ramyagovind13** [comment]
Looks good to me!

**Review by ramyagovind13 — APPROVED** [review]

**Pull request merged**